app/build.gradle

```diff
@@ -58,16 +58,15 @@ dependencies {
     // this syntax doesn't work for compiling .aar files, so those have to be loaded manually
     implementation fileTree(include: '*.jar', exclude: 'regexp-me.jar', dir: 'libs')
     implementation(name: 'htmlspanner-custom', ext: 'aar')
-    implementation(name: 'android-zebra-interface-1.1', ext: 'aar')
+    implementation 'com.github.dimagi:zebra-print-android:v1.2'
     implementation 'com.simprints:LibSimprints:1.0.12'
     implementation 'androidx.multidex:multidex:2.0.1'
     implementation 'androidx.cardview:cardview:1.0.0'
     implementation 'androidx.recyclerview:recyclerview:1.1.0'
     implementation 'androidx.legacy:legacy-support-v4:1.0.0'
     implementation 'androidx.legacy:legacy-preference-v14:1.0.0'
     implementation 'androidx.gridlayout:gridlayout:1.0.0'
-    implementation 'com.madgag.spongycastle:core:1.54.0.0'
-    implementation 'com.madgag.spongycastle:prov:1.54.0.0'
+    implementation 'org.bouncycastle:bcprov-jdk15on:1.65'
```

Comment:
The latest version of bouncycastle `1.66` is currently giving a build error: `Failed to transform org.bouncycastle/bcprov-jdk15on/1.66 using Jetifier. Reason: IllegalArgumentException, message: Unsupported class file major version 59.` So using `1.65`.